docs/make.jl

```diff
@@ -46,6 +47,7 @@ end
 #####
 
 example_pages = [
+    "Bickley_jet_shallow_water.jl"     => "generated/Bickley_jet_shallow_water.md", 
```

Comment:
Change this to:
```Julia
"Bickley Jet with Shallow Water" => "generated/Bickley_jet_shallow_water.md",
```

Reply:
What you have on the LHS of `=>` is the title that appears in the TOC sidebar on the left.

Reply:
Great and thanks for clarifying